core/templates/domain/skill/skill-creation-backend-api.service.ts

```diff
@@ -19,6 +19,7 @@
 import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
+import { ImageLocalStorageService } from 'services/image-local-storage.service';
```

Comment:
```suggestion

import { ImageLocalStorageService } from 'services/image-local-storage.service';
```

lets keep library imports separate

Reply:
done